teammapper-frontend/angular.json

```diff
@@ -87,7 +87,8 @@
               "buildTarget": "teammapper:build:production"
             },
             "development": {
-              "buildTarget": "teammapper:build:development"
+              "buildTarget": "teammapper:build:development",
```

Comment:
disablehostcheck should be removed I guess?

Reply:
No this is actually the correct place to put it, since this won't affect production and we won't have a billion angular.json changes in every PR just because of me :D